packages/qwik/src/optimizer/src/plugins/vite.ts

```diff
@@ -649,8 +649,8 @@ export function qwikVite(qwikViteOpts: QwikVitePluginOptions = {}): any {
       }
 
       if (!qwikViteOpts.csr) {
+        const opts = qwikPlugin.getOptions();
         const plugin = async () => {
-          const opts = qwikPlugin.getOptions();
```

Comment:
I think this should stay like it was, the options can change before this plugin is called